backend/src/main/java/capstone/backend/model/dto/order/OrderToSupplierDTO.java

```diff
@@ -1,5 +1,12 @@
 package capstone.backend.model.dto.order;
 
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder
```

Comment:
are you sure that you don't want to add the NoArgsConstructur and AllArgsC. ? Just to be save :-)

Reply:
makes sense